samples/customHandler/config.capnp

```diff
@@ -0,0 +1,35 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const helloWorldExample :Workerd.Config = (
+  services = [
+    ( name = "main", worker = .helloWorld ),
+    ( name = "server", worker = .server ),
+  ],
+  sockets = [ ( name = "http", address = "*:8080", http = (), service = "main" ) ],
+);
+
+const helloWorld :Workerd.Worker = (
+  modules = [
+    ( name = "worker", esModule = embed "worker.js" ),
+    ( name = "jsonRpc", esModule = embed "jsonRpc.js" )
```

Comment:
So the wrapper module is bundled with the worker? Don't we want these wrappers to be bundled with the runtime, like we're doing with node compat modules currently?

Reply:
not necessary. `wrapWith` just needs to be importable. It could be `cloudflare:email` and come from a built in.

Reply:
But I think we want these wrapper modules to be hidden modules, not directly importable by the application.